components/email/stylesheets/editorial_signup.styl

```diff
@@ -115,5 +115,81 @@
     position absolute
     background-image linear-gradient(to right, rgba(0,0,0,1) 0%, rgba(0,0,0,0) 65%, rgba(0,0,0,1) 100%)
 
+.articles-es-cta--banner
+  @extend .articles-es-cta
+  background-color #F8F8F8
```

Comment:
I don't see #f8f8f8 in the style guide, but is there a similar one we could use instead from [here?](https://www.artsy.net/style-guide/colors)